api/views.py

```diff
@@ -21,69 +21,46 @@
 #  VulnerableCode is a free software code scanning tool from nexB Inc. and others.
 #  Visit https://github.com/nexB/vulnerablecode/ for support and download.
 
-from rest_framework import serializers
-from rest_framework import status
-from rest_framework import viewsets
-from rest_framework.response import Response
+import json
+
+from django.http import HttpResponse
+
+from rest_framework.viewsets import ReadOnlyModelViewSet
+from rest_framework.serializers import ValidationError
 
 from packageurl import PackageURL
 
-from vulnerabilities.models import Package
-from vulnerabilities.models import PackageReference
-from vulnerabilities.models import Vulnerability
-from vulnerabilities.models import VulnerabilityReference
-
-
-class PackageReferenceSerializer(serializers.ModelSerializer):
-    class Meta:
-        model = PackageReference
-        fields = [
-            'repository',
-            'platform',
-            'name',
-            'version',
-        ]
-
-
-class VulnerabilityReferenceSerializer(serializers.ModelSerializer):
-    class Meta:
-        model = VulnerabilityReference
-        fields = [
-            'source',
-            'reference_id',
-            'url',
-        ]
-
-
-class VulnerabilitySerializer(serializers.ModelSerializer):
-    references = VulnerabilityReferenceSerializer(source='vulnerabilityreference_set', many=True)
-
-    class Meta:
-        model = Vulnerability
-        fields = [
-            'summary',
-            'cvss',
-            'references',
-        ]
-
-
-class PackageSerializer(serializers.ModelSerializer):
-    vulnerabilities = VulnerabilitySerializer(many=True)
-    references = PackageReferenceSerializer(source='packagereference_set', many=True)
-
-    class Meta:
-        model = Package
-        fields = [
-            'name',
-            'version',
-            'package_url',
-            'vulnerabilities',
-            'references',
-        ]
-
-
-class PackageViewSet(viewsets.ReadOnlyModelViewSet):
-    queryset = Package.objects.all()
+from api import cve_search
+from api.serializers import PackageSerializer
+from core import models
+
+
+def package(request, name):
+    """
+    Queries the cve-search api with just
+    a package name.
+    """
+    raw_data = cve_search.data_cve_circl(name=name)
+    fields_names = ['id', 'summary', 'cvss']
+    extracted_data = cve_search.extract_fields(raw_data, fields_names)
+
+    return HttpResponse(json.dumps(extracted_data))
+
+
+def package_version(request, name, version):
+    """
+    Queries the cve-search api with a package
```

Comment:
Do we really want to call this cve-search?
This is a more generic vulnerability search than just CVE so I would refrain to use CVE as a term unless we are specifically dealing with the CVE... since many vulnerability may not have a CVE in the future.

Reply:
My question here would rather be do we really want to have this endpoint at all. It is a proxy to http://cve.circl.lu/api/search/, so the name is accurate but I'm not sure what the purpose is.
I just kept this mostly as is (module some file renaming) to keep the scope of the PR small.